src/main/kotlin/com/procurify/flagcounter/FlagCounter.kt

```diff
@@ -76,11 +79,16 @@ class FlagCounter(
             this.postMessage(message).mapLeft { throw Exception(message) }
 
     /**
-     * Formats a [totalFlagCount] into a message to be sent by the messager
+     * Formats a message about the total flag count in the system based on the [teamFlags]
      */
-    private fun formatTotalCountMessage(totalFlagCount: Int): String {
+    private fun formatTotalCountMessage(teamFlags: List<FlagDetail>): String {
```

Comment:
minor: If we follow the principle of least knowledge here and pass `flagSize` and `medianAgeInDays` as arguments, this function may be easier to test as we then do not need to mock/instantiate the teamFlags object to pass in.